build.gradle

```diff
@@ -33,6 +33,9 @@ sourceSets {
         java {
             srcDir 'cases/src'
             srcDir 'backend/src'
+            srcDir 'api'
+            srcDir 'core'
+            srcDir 'modern'
```

Comment:
@phillipm I think that might make the Ant build pretty rough for this project? Not 100% sure.